src/rpc/client/channelOwner.ts

```diff
@@ -74,27 +70,21 @@ export abstract class ChannelOwner<T extends Channel = Channel, Initializer = {}
   }
 
   _dispose() {
-    assert(this._isScope);
-
     // Clean up from parent and connection.
     if (this._parent)
       this._parent._objects.delete(this._guid);
     this._connection._objects.delete(this._guid);
 
     // Dispose all children.
-    for (const [guid, object] of [...this._objects]) {
-      if (object._isScope)
-        object._dispose();
```

Comment:
Trying to make sense of it. What happens when I remove the `context`, why does the `page` dispose? I.e. what removes all the element handles from the page when context is disposed?